src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4832nocstylearray/InputNoCstyleArrays.java

```diff
@@ -22,38 +21,34 @@ public static void mainC(String aCStyle[]) // violation 'Array brackets at illeg
     //                    'Array brackets at illegal position.'
   }
 
+  /** some javadoc. */
   public class Test {
     public Test[] variable; // ok
 
     public Test[] getTests() {
       return null;
     }
 
-    public Test[] getNewTest() // ok
-    {
+    public Test[] getNewTest() { // ok
       return null;
     }
 
-    public Test getOldTest()[] // violation 'Array brackets at illegal position.'
-    {
+    public Test getOldTest()[] { // violation 'Array brackets at illegal position.'
       return null;
     }
 
-    public Test getOldTests()[][]
-    // 2 violations above:
+    // 2 violations 3 lines below:
     //                    'Array brackets at illegal position.'
     //                    'Array brackets at illegal position.'
-    {
+    public Test getOldTests()[][] {
       return null;
     }
```

Comment:
moved the comment outside of method so we don't cross the limit of `minLineCount`:

https://github.com/checkstyle/checkstyle/blob/2057ac975e1c9ad48fdad077afc0f75ae6bb23bc/src/main/resources/google_checks.xml#L338-L343